### PR TITLE
README.md: Editing for clarity

### PR DESCRIPTION
**Pull request by jeremiah**
Changes to make building an image clearer (hopefully.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PELUX baseline manifests
 =========================
-This is the manifest repository for building the PELUX baseline for various
-hardware targets using Yocto
+This manifest repository is used for building the PELUX baseline for various
+hardware targets using the Yocto build tool.
```
Comment (breakreturn) [inline README.md]: I don't feel it's more correct like this Yocto is usually referred to as more than the tool used for building. Maybe "using the Yocto project"?

File: README.md [hunk 2]
```diff
@@ -112,13 +112,13 @@ repo init -u https://github.com/Pelagicore/pelux-manifests.git -m <manifest> -b
 repo sync
 ```
 
-When done fetching the sources, create a build directory and set up bitbake.
-TEMPLATECONF tells the `oe-init-build-env` script which path to fetch
-configuration samples from. Note that the example below get the template
-configuration for the Intel BSP, adapt the path according to your current BSP.
+When 'repo sync' has finished fetching the sources, the next step is to create a 'build' directory and set up bitbake.
```
Comment (DunderRoffe) [inline README.md]: I would prefer ` (back ticks) over ' which puts the content in a box making it clearer that you are talking about a command.

File: README.md (hunk 1 above)
Comment (GordanM) [reply README.md]: +1

File: README.md [hunk 3]
```diff
@@ -112,13 +112,13 @@ repo init -u https://github.com/Pelagicore/pelux-manifests.git -m <manifest> -b
...
+The TEMPLATECONF envorinment setting tells the `oe-init-build-env` script which path to fetch
```
Comment (tobsan) [inline README.md]: environment*

File: README.md [hunk 4]
```diff
@@ -138,14 +140,12 @@ information about how to use a built image with the targets, see
 [Getting started](getting-started.md).
 
 ### Pelux Intel
-Reference instance for the Intel i7 x86 platform. Examples of boards using this
-architecture include:
+Reference instance for the Intel i7 x86 platform. 
 
-* Intel NUC
-* Minnowboard
+* [Intel NUC](https://en.wikipedia.org/wiki/Next_Unit_of_Computing)
```
Comment (tobsan) [inline README.md]: Please keep the Minnowboard here, as it, from a Yocto perspective, uses the i7 platform. Perhaps we can change the wording to just say generic Intel amd64 or something.

File: README.md (hunk 1 above)
Comment (jeremiah) [reply README.md]: "using the Yocto project" is better. I'll change.

Comment (jeremiah) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (jeremiah) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (jeremiah) [reply README.md]: Done

**Pull request closed without merging**